modules.d/91tpm2-tss/module-setup.sh

```diff
@@ -52,6 +52,7 @@ install() {
         {"tls/$_arch/",tls/,"$_arch/",}"libtss2-tcti-swtpm.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libtss2-tctildr.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libcryptsetup.so.*" \
+        {"tls/$_arch/",tls/,"$_arch/",}"/cryptsetup/libcryptsetup-token-systemd-tpm2.so.*" \
```

Comment:
Both of these have end in `.so` not `.so.*`. I don't have a file matching the glob ending in `.so.*`:

```
 $ find /usr/lib64/cryptsetup
/usr/lib64/cryptsetup
/usr/lib64/cryptsetup/libcryptsetup-token-systemd-tpm2.so
/usr/lib64/cryptsetup/libcryptsetup-token-systemd-fido2.so
/usr/lib64/cryptsetup/libcryptsetup-token-systemd-pkcs11.so
```

Reply:
Which distribution do you use?

Reply:
Gentoo

Reply:
It's the same for me on Arch Linux:

```
$ find /usr/lib/cryptsetup/
/usr/lib/cryptsetup/
/usr/lib/cryptsetup/libcryptsetup-token-systemd-fido2.so
/usr/lib/cryptsetup/libcryptsetup-token-systemd-pkcs11.so
/usr/lib/cryptsetup/libcryptsetup-token-systemd-tpm2.so
```

Reply:
Updated please test

Reply:
LGTM